builder/dockerfile/internals_test.go

```diff
@@ -22,7 +23,7 @@ func TestEmptyDockerfile(t *testing.T) {
 
 	createTestTempFile(t, contextDir, builder.DefaultDockerfileName, "", 0777)
 
-	readAndCheckDockerfile(t, "emptyDockerfile", contextDir, "", "the Dockerfile (Dockerfile) cannot be empty")
+	readAndCheckDockerfile(t, "emptyDockerfile", contextDir, "", `the Dockerfile \(Dockerfile\) cannot be empty`)
```

Comment:
This change of meaning of this sting is easy to break. Could we either pass a regexp now, or add a special wrapper that can be called when regexp check is needed.